.github/workflows/run_local_tests.yml

```diff
@@ -129,7 +129,7 @@ jobs:
       - name: 'Docker clean containers and images after the test'
         if: always()
         run: |
-          for i in `docker ps -a | awk '{ print $1 } ' | grep -v CONTAINER`; do docker stop $i && docker rm $i; done
+          docker stop $(docker ps -a -q) && docker rm $(docker ps -a -q) && docker volume rm $(docker volume ls -q)
```

Comment:
Not sure why this would speed it up ... did you time it to test? In your implementation docker ps is called three times, rather than onces, and you are doing a docker volume remove per container, when later we explicitly prune all volumes including those that are dangling?